docs/self-managed/setup/guides/connect-to-an-oidc-provider.md

```diff
@@ -165,8 +176,8 @@ global:
       identity:
         clientId: <Client ID from Step 2>
         existingSecret: <Client secret from Step 5>
-        audience: <Audience from Step 1>
-        initialClaimName: <Initial claim name if not using the default "oid">
+        audience: <Audience from Step 2>
```

Comment:
❓ here we use `step 2` but above we use `step 3`, both points are related IMO so maybe step 2 and 3 could be combined?

Reply:
Thank you for catching this - I went ahead and renumbered a bunch of these inaccurate steps!